earthpy/plot.py

```diff
@@ -74,9 +75,7 @@ def colorbar(mapobj, size="3%", pad=0.09):
 def plot_bands(
     arr, cmap="Greys_r", figsize=(12, 12), cols=3, title=None, extent=None
 ):
-    """Plot each layer in a raster stack read from rasterio in
-    (band, row , col) order as a numpy array. plot_bands will create an
-    individual plot for each band in a grid.
+    """Individually plot each band from rasterio order(band, row , col) stack.
```

Comment:
`Plot each band  in a numpy array in its own axis. Assumes band order (band, row, col)`